WooCommerce/src/main/kotlin/com/woocommerce/android/ui/main/MainActivity.kt

```diff
@@ -256,6 +257,18 @@ class MainActivity : AppCompatActivity(),
         initFragment(null)
     }
 
+    /**
+     * Called when the user switches sites - reset the fragments and tell the dashboard to refresh
+     */
+    override fun resetSelectedSite() {
+        bottomNavView.reset()
+        val dashboard = bottomNavView.getFragment(DASHBOARD) as DashboardFragment?
```

Comment:
The return type of `bottomNavView.getFragment()` is not nullable so you don't need to check for nullability.
```suggestion
        val dashboard = bottomNavView.getFragment(DASHBOARD) as DashboardFragment
```

Reply:
Good call, corrected in d62560a